runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -12491,10 +12497,10 @@ TR::Register *J9::Power::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::C
 
    if (postP10Copy)
       {
-      TR::addDependency(deps, NULL, TR::RealRegister::vsr8, TR_VSX_VECTOR, cg);
-      TR::addDependency(deps, NULL, TR::RealRegister::vsr9, TR_VSX_VECTOR, cg);
-      TR::addDependency(deps, NULL, TR::RealRegister::vsr32, TR_VSX_VECTOR, cg);
-      TR::addDependency(deps, NULL, TR::RealRegister::vsr33, TR_VSX_VECTOR, cg);
+      TR::addDependency(deps, vec0Reg, TR::RealRegister::vsr8, TR_VSX_VECTOR, cg);
+      TR::addDependency(deps, vec1Reg, TR::RealRegister::vsr9, TR_VSX_VECTOR, cg);
+      TR::addDependency(deps, fp1Reg, TR::RealRegister::vsr32, TR_VSX_VECTOR, cg);
+      TR::addDependency(deps, fp2Reg, TR::RealRegister::vsr33, TR_VSX_VECTOR, cg);
       }
```

Comment:
there are non-p10 paths having the same issue, right?  i.e.  created virtual registers are not in named variables. you should not look at only p10-path.

a simpler fix (for all cases) is to make sure all registers created in this scope are on dependency, and registers carried back in Node(s) can be excluded later when calling deps->stopUsing..   then, you only need to insert two calls to that (one for inlined ICF case and another one for the remaining case) and eliminate all reg->stopUsing